From ef4b8a7341120c79800c92be7541dd7b2bfef1b2 Mon Sep 17 00:00:00 2001
From: Demon000 <demonsingur@gmail.com>
Date: Thu, 11 Jan 2018 10:29:24 +0200
Subject: [PATCH 10/25] legacy: add back ipacm rules

Removed in 42da8c6f03f8abb3ec5df68ec263cea33f9fb59c,
needed for old blobs.

Also, give it access to its own files.

Change-Id: I25a34d5398076f15cf0e7a33605f8c17bb106211
---
 legacy-common/ipacm.te | 5 +++++
 1 file changed, 5 insertions(+)
 create mode 100644 legacy-common/ipacm.te

diff --git a/legacy-common/ipacm.te b/legacy-common/ipacm.te
new file mode 100644
index 0000000..07fe99b
--- /dev/null
+++ b/legacy-common/ipacm.te
@@ -0,0 +1,5 @@
+# Allow netfilter_socket creation
+allow ipacm self:netlink_netfilter_socket create_socket_perms_no_ioctl;
+
+allow ipacm ipacm_data_file:dir w_dir_perms;
+allow ipacm ipacm_data_file:file create_file_perms;
-- 
2.7.4

